android/src/main/java/com/brentvatne/react/ReactVideoPackage.java

```diff
@@ -1,14 +1,15 @@
 package com.brentvatne.react;
 
 import com.brentvatne.exoplayer.DefaultReactExoplayerConfig;
-import com.brentvatne.exoplayer.ReactExoplayerConfig;
 import com.brentvatne.exoplayer.ReactExoplayerViewManager;
 import com.facebook.react.ReactPackage;
-import com.facebook.react.bridge.JavaScriptModule;
 import com.facebook.react.bridge.NativeModule;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.uimanager.ViewManager;
 
+import com.brentvatne.exoplayer.ReactExoplayerConfig;
```

Comment:
these changes are not necessary

Reply:
I think we need to import that `com.brentvatne.exoplayer.ReactExoplayerConfig`?
I can see compile error when I removed that line.